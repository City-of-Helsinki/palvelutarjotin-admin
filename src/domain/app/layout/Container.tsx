import classnames from 'classnames';
import React, { FunctionComponent } from 'react';

import styles from './container.module.scss';

const Container: FunctionComponent<{
  className?: string;
}> = ({ children, className }) => {
  return (
    <div className={classnames(styles.container, className)}>{children}</div>
  );
};

export default Container;
