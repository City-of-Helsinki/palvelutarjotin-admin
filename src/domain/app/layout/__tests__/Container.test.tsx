import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as React from 'react';

import Container from '../Container';

it('Container matches snapshot', () => {
  const container = shallow(<Container>TEST CONTENT</Container>);
  expect(toJson(container)).toMatchSnapshot();
});
