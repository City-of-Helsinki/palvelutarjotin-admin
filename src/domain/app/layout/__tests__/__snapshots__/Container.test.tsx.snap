// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container matches snapshot 1`] = `
<div
  className="container"
>
  TEST CONTENT
</div>
`;
